ReleaseProcess.md

```diff
@@ -1,57 +1,87 @@
+7. Delete the scale cluster when you are done validating (it's in AME)
 
 
-# 2. Perf and scale testing
+# 2. Production Image to MCR CN for Azure China Cloud
 
-Deploy latest omsagent yaml with release candidate agent image in to supported k8s versions and validate all the critical scenarios. In perticular, throughly validate the updates going as part of this release and also make sure no regressions. If  this passes, deploy onto scale cluster and  validate perf and scale aspects. Scale cluster in AME cloud and co-ordinate with agent team who has access to this cluster to deploy  the release candiate onto this cluster.
+No action required here, images are automatically synched to MCR CN from Public cloud MCR.
 
-# 3. Production Image to MCR CN for Azure China Cloud
+# 3. Release of the agent
 
-Image automatically synched to MCR CN from Public cloud MCR.
+## 3.1. AKS
+(Last updated 5/18/2021)
 
-# 4. Release of the agent
+1. Update the image in the AKS Agent Baker (use [this PR as a reference for linux](https://github.com/Azure/AgentBaker/pull/775) and [this PR as a reference for windows](https://github.com/Azure/AgentBaker/pull/813)). The AKS agent is built every Friday at 8:00 PM  (as of 4/23/2021, may change in the future)
+2. Update the agent version in the AKS RP ([use this PR as a reference](https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp/pullrequest/4425425)). Make this PR on Monday following the Agent Baker PR.
 
-## AKS
+   Most of the files in the reference PR are auto-generated templates used for unit tests. They can be manually updated or auto-generated. The auto-generation tool doesn't always work though, sometimes it generates results that fail AKS tests. It is known to work with helm 2.12.3, please update this section if you get it working.
+   ```
+   # from the base directory of the aks-rp repository
+   cd ccp\charts\kube-control-plane\tests
+   Set RENDER_SNAPSHOTS=true
+   go test .
+   ```
 
-- Refer to internal docs for the release process and instructions.
+3. update [this dashboard](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/dashboard/arm/subscriptions/13d371f9-5a39-46d5-8e1b-60158c49db84/resourcegroups/dashboards/providers/microsoft.portal/dashboards/011897c2-1ea6-4e21-bc8e-2c835fe40024). About half way down is a chart called Latest "LINUX agent (ciprod________) - Cluster count" and "Latest Windows agent (win-ciprod________) - Cluster count". Remove the oldest pair of charts and add new ones for the agent version about to roll out. This is so we can track the rollout.
```

Comment:
same case here .. this is internal ms link and we shouldnt put in open source code. we just  to add  this our internal CI wiki and just say refer internal CI wiki.